### PR TITLE
[15.0][MIG] base_custom_info

### PR DESCRIPTION
**Pull request by Rad0van**
Fairly standard migration. only had to deal with access to ir.model*, new style assets in manifest and some cosmetic details regarding string formatting and exception handling. Tested and seems to work as in previous versions.

### CODE REVIEW

File: base_custom_info/__manifest__.py [hunk 1]
```diff
@@ -7,10 +7,9 @@
     "name": "Base Custom Info",
     "summary": "Add custom field in models",
     "category": "Tools",
-    "version": "14.0.1.0.1",
+    "version": "15.0.1.0.1",
```
Comment (sebalix) [inline base_custom_info/__manifest__.py]: You can reset the version to `15.0.1.0.0`

Comment (dalonsod) [reply base_custom_info/__manifest__.py]: @Rad0van locally tested, LGTM.
If manifest version is changed as @sebalix sugguests I'll approve it

Comment (Rad0van) [reply base_custom_info/__manifest__.py]: Done.

**Review by dalonsod — APPROVED** [review]

File: base_custom_info/__manifest__.py (hunk 1 above)
Comment (dalonsod) [reply base_custom_info/__manifest__.py]: @sebalix could you approve this PR?

**Review by bosd — APPROVED** [review]
Very quick code review (changes only)
Would love to see this one quickle merged, as there are PR's depending on this one.

**Comment by dalonsod** [comment]
@Rad0van could you check #2705 and cherry-pick it if is finally needed?

**Comment by Rad0van** [comment]
> @Rad0van could you check #2705 and cherry-pick it if is finally needed?

No problem. But should that be part of the migration commit or a standalone one?

**Comment by dalonsod** [comment]
In order to preserve commit authorship, the fix should be cherry-picked, and not merged with migration commit.

**Comment by dalonsod** [comment]
Tested with recently added fix, it works in v15 as well. I won't approve this... again 😝

@etobella could you review?

**Comment by Rad0van** [comment]
As this has been merged can you @sebalix @dalonsod @pedrobaeza @bosd have a look at https://github.com/OCA/product-attribute/pull/1091 ?  Thank you.

**Pull request merged**